templates/vue/src/router.js

```diff
@@ -20,6 +20,14 @@ const router = new VueRouter({
   routes,
 })
 
+// sourced from https://stackoverflow.com/questions/58634914/getting-navigation-duplicated-error-while-the-route-is-being-replaced-on-vue-rou

```

Comment:
Can you remove everything after the question number in the URL? The last part is not needed for the URL to work.